src/types/dto/workflowStates.dto.ts

```diff
@@ -1,6 +1,7 @@
 import { z } from 'zod'
 
 export const WorkflowStateTypeSchema = z.enum(['backlog', 'unstarted', 'started', 'completed', 'cancelled'])
+export type WorkflowStateType = z.infer<typeof WorkflowStateTypeSchema>
```

Comment:
We could simply name it `WorkflowState`

Reply:
Removed in be7bf4d309a4944872b6c2f5f42cea7ec1aeee8d commit